packages/pigeon/unit_tests/ios_unit_tests/.gitignore

```diff
@@ -0,0 +1,37 @@
+# Miscellaneous
```

Comment:
The distinction between the test/ and unit_tests/ directory is not clear. Maybe just test/ and platform_test/? Since the things in test/ are unit tests too.

Reply:
done